core/webui-explorer/src/main/java/org/exoplatform/ecm/webui/component/explorer/UIDocumentAutoVersionForm.java

```diff
@@ -166,8 +166,8 @@ public void execute(Event<UIDocumentAutoVersionForm> event) throws Exception {
               }
               destPath = newDestPath;
               copyTitle = destPath.substring(destPath.lastIndexOf("/") + 1, lastDotIndex) + "(" + (fileIndex + 1 ) + ")" + destPath.substring(lastDotIndex);
-
             }
+
```

Comment:
Please Remove theses 2 modifications

Reply:
I removed it

Reply:
you still have a line remove. Please check the diff and readd it

Reply:
I check it and I correct it